examples_test.go

```diff
@@ -430,9 +440,12 @@ func ExampleDevice_Assertion_hmacSecret() {
 	if testVector.Secret != hex.EncodeToString(assertion.HMACSecret) {
 		log.Fatalf("Expected %s", testVector.Secret)
 	}
+
+	// Output:
+	//
 }
 
-func ExampleDevice_DeleteCredential() {
+func Dont_ExampleDevice_DeleteCredential() {
```

Comment:
That one should not run this.  :)

This is mainly to avoid someone running the entire file and getting all sorts of damaging operations done to their Yubikeys. I've done that in a few other tests here before to avoid outright deleting them.

Reply:
A better explanation is that the Dont_ makes it not a test. If you want to run it, you have to remove the Dont prefix.

Reply:
You could make them tests and require an environment variable to be set and skip otherwise (or something similar). In any case, I'd comment what is the intent. Comments in the code are easier to find than a PR comment.

Reply:
I don't think it's worth spending too much time fiddling with this, apart from making sure it doesn't do anything too destructive. These all came from upstream at some point and I don't typically patch those merges on the spot, unless they are broken.

Reply:
> You could make them tests and require an environment variable to be set and skip otherwise

They already have that.

>  it would be clearer if we added a t.Skip

There's no `testing.T` in Example tests.

I'm trying not to deviate too much from the upstream, for whatever that is worth it, while avoiding a "miss-command" deleting important credentials from plugged Yubikeys. As stated before I don't think it's worth chasing this too much.

Reply:
Dont_ tests deleted, PTAL.